### PR TITLE
Update go and node version for building nitro locally

### PR DESCRIPTION
**Pull request by amsanghi**

### CODE REVIEW

File: arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md [hunk 1]
```diff
@@ -109,15 +109,15 @@ sudo mkdir -p /usr/local/bin
...
+### Step 5. Configure node [18.20.3](https://github.com/nvm-sh/nvm)
 
 #### For Debian/Ubuntu
 
 ```bash
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
 source "$HOME/.bashrc"
-nvm install 16.19
-nvm use 16.19
+nvm install 18.20.3
```
Comment (eljobe) [inline arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md:119]: Do we need to pin to the specific 18.20.3 version, or would it be sufficient to use "18" here?

Comment (amsanghi) [reply arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md]: Good point, updated, thanks !

**Review by eljobe — APPROVED** [review]
LGTM

File: arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md [hunk 2]
```diff
@@ -109,15 +109,15 @@ sudo mkdir -p /usr/local/bin
 sudo ln -s  /opt/homebrew/opt/llvm/bin/wasm-ld /usr/local/bin/wasm-ld
 ```
 
-### Step 5. Configure node [16.19](https://github.com/nvm-sh/nvm)
+### Step 5. Configure node [18](https://github.com/nvm-sh/nvm)
 
 #### For Debian/Ubuntu
 
 ```bash
```
Comment (anegg0) [inline arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md:116]:
```suggestion
```shell
```

File: arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md [hunk 3]
```diff
@@ -163,8 +163,8 @@ brew install bison
...
```
Comment (anegg0) [inline arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md:163]:
```suggestion
```shell
```

**Review by anegg0 — APPROVED** [review]
Aside from a nit (we switched code blocks' flags to `shell`), lgtm!

**Pull request merged**